app/main/forms/work_order_data.py

```diff
@@ -0,0 +1,8 @@
+questions = {
+    'number': {
+        'heading': 'Work Order number',
+        'summary': 'This is a seller-supplied work order number',
+        'label': 'Work Order number',
+        'message': 'You must provide a Work Order number'
```

Comment:
It would be easier for people to understand this without reading the main Python code if it were called `error_message`.